src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseService.java

```diff
@@ -506,6 +508,13 @@ public ProgrammingExercise updateProblemStatement(ProgrammingExercise programmin
         return updatedProgrammingExercise;
     }
 
+    public void validateNumberOfTasks(ProgrammingExercise programmingExercise) {
```

Comment:
I don’t interpret Stephan’s comment in the linked issue as ‘There should be a hard limit.’, I rather understood it as ‘From an educational/didactic perspective, it does not make sense to have that many tasks.’.

If there is no technical limitation why there should be an upper bound (I don’t think there is since it was not a problem before), the instructor should be able to create any number of tasks without a hard limit they want to even if that is didactically not desirable.
Showing a warning for >15 should be enough to make the instructor think about it again if they really want to do it, and then they are still free to decide that they _really_ want that many tasks.

Reply:
Done as you said